server/src/main/scala/com/cloudera/livy/server/client/ClientSessionServlet.scala

```diff
@@ -53,17 +60,47 @@ class ClientSessionServlet(sessionManager: SessionManager[ClientSession, CreateC
     }
   }
 
+  jpost[Any]("/:id/upload-jar") { req =>
```

Comment:
Ah, ok. I was wondering how to test this I guess I will need to set the body to Unit and then somehow set the file attachments.